paper.md

```diff
@@ -0,0 +1,87 @@
+---
+title: 'scikit-COSMO: A Toolbox of Machine Learning Methods for Materials Science'
+tags:
+    - Python
+authors:
+    - name: Rose K. Cersonsky ^ [\url{mailto: rose.cersonsky@epfl.ch}]
+    orcid: 0000-0003-4515-3441
+    affiliation: 1
+    - name: Guillaume Fraux
+    orcid: 0000-0003-4824-6512
+    affiliation: 1
+    - name: Sergei Kliavinek
+    orcid: 0000-0001-8326-325X
+    affiliation: 1
+    - name: Alexander Goscinski
+    orcid: 0000-0001-8076-215X
+    affiliation: 1
+    - name: Benjamin A. Helfrecht
+    orcid: 0000-0002-2260-7183
+    affiliation: 1
+    - name: Michele Ceriotti ^ [\url{mailto: michele.ceriotti@epfl.ch}]
+    orcid: 0000-0003-2571-2832
+    affiliation: 1
+affiliations:
+    - name: Laboratory of Computational Science and Modeling(COSMO), École Polytechnique Fédérale de Lausanne(EPFL), Lausanne, Switzerland
+    index: 1
+date: 03 May 2021
+bibliography: paper.bib
+
```

Comment:
maybe this empty line is causing the metadata issue with the preview?

Reply:
The date line?

Reply:
no, the empty one after bibliography